content/en/docs/reference/glossary/istio-resource.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Istio Resource
```

Comment:
Is this too generic?  I would have thought "Istio resource" could apply to any API object created against an Istio CRD.

(Otherwise LGTM)

Reply:
Let me remove that one. I could have sworn I saw somewhere where the docs were simply referring to it as "your Istio resource" but now I'm not seeing it. I agree that it's too generic.